bpf/node_config.h

```diff
@@ -27,6 +27,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define UNMANAGED_ID 3
 #define HEALTH_ID 4
 #define INIT_ID 5
+#define LOCAL_NODE_ID 6
 #define REMOTE_NODE_ID 6
```

Comment:
It could, but I don't like it. This file is an example node config header that is used for compiling bpf via make. In actual use Cilium Agent will always write this header file based on the agent runtime configuration, and the values can be different. Defining on in the terms of another would be misleading here.